src/Generator/PropertyGenerator.php

```diff
@@ -179,19 +213,15 @@ public function setDefaultValue(
         return $this;
     }
 
-    /**
-     * @return ?PropertyValueGenerator
-     */
-    public function getDefaultValue()
+    public function getDefaultValue(): ?PropertyValueGenerator
     {
         return $this->defaultValue;
     }
 
     /**
      * @throws Exception\RuntimeException
-     * @return string
      */
-    public function generate()
+    public function generate(): string
```

Comment:
We can't add the native return type declaration here

Reply:
Reverted and added psalm non-empty-string